src/containers/ProjectsList.js

```diff
@@ -1,8 +1,40 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import { Header } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { fetchProjects } from '../actions/getProjectsAction'
+import Paginate from '../components/Paginate'
+import PaginationLinks from '../components/PaginationLinks'
 
 export class ProjectsList extends Component {
+  state = { firstPage: true, lastPage: true }
+
+  handlePageSelect = selectedPage => e => {
```

Comment:
Isn't the did going to be the same in all things paginated? Shouldn't it be in the pagination container?

Reply:
Not sure what you mean by this... are you asking if the `handlePageSelect` should live in the PaginationLinks component or Paginate component since it is being reused by many different components? 
The `UsersList` and `ProjectsList`  container are both using the `handlePageSelect` method, but they are checking for different things and they are setting the state, so I think it wouldn't be practical to extract this method from the containers. 
If you look at `PR` https://github.com/AgileVentures/WebsiteOne-FE/pull/45 it gives a better sense of this method with the latest changes.